web/src/utils/factory-management/buildings.ts

```diff
@@ -21,13 +21,13 @@ export const calculateBuildingRequirements = (factory: Factory, gameData: DataIn
       }
 
       const buildingData = recipe.building
-      const buildingCount = (product.amount / productInRecipe.perMin).toFixed(3)
+      const buildingCount = product.amount / productInRecipe.perMin
 
       product.buildingRequirements = {
         name: buildingData.name,
         amount: buildingCount,
-        powerPerBuilding: Number(buildingData.power),
-        totalPower: (parseInt(buildingData.power) * parseInt(buildingCount)).toFixed(2),
+        powerPerBuilding: buildingData.power,
+        totalPower:  buildingData.power * buildingCount,
```

Comment:
Nitpick but extra space?

Reply:
Do you know how to nitpick the space right in here? You can do it with the add a suggestion button, and then it's easy to fix right in the PR. 
```suggestion
        totalPower: buildingData.power * buildingCount,
```

Reply:
Ahh yes I keep forgetting that's a thing 😄